wicket/src/ui/panes/overview.rs

```diff
@@ -1130,7 +1130,9 @@ fn append_caboose(
         git_commit,
         // Currently `name` is always the same as `board`, so we'll skip it.
```

Comment:
Is this comment still accurate? I know it's true for production images, but could this bit of code see non-prod builds with different names?

Reply:
I think you are correct and it could be helpful to confirm this via wicket.